torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -4,28 +4,28 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 namespace {
-py::object module_;
-py::object runUDFFunction_;
-py::object loadResultFunction_;
+py::object runUDFFunction_ = py::none();
```

Comment:
Is this intentional to put changes in `python_rpc_handler.cpp` into this PR?

Reply:
yes, otherwise tests will fail